recipe/meta.yaml

```diff
@@ -9,29 +9,25 @@ source:
   sha256: 7682dc8afb30297001674575ea00d1814d808d6a36af415a82bd481d37ba7b8e
 
 build:
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  number: 2
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
-    - python
+    - python >=3.8
     - pip
   run:
-    - python
-    - colorama  # [win]
-    - importlib-metadata  # [py<38]
+    - python >=3.8
+    - colorama
```

Comment:
@isuruf what is the best choice here? Just ship colorama on not win too or do we have a better option?